src/compas/geometry/primitives/plane.py

```diff
@@ -204,7 +206,7 @@ def from_three_points(cls, a, b, c):
         a = Point(*a)
         b = Point(*b)
         c = Point(*c)
-        normal = Vector.cross(b - a, c - a)
+        normal = Vector(*cross_vectors(b - a, c - a))
```

Comment:
`b - a` is a `Vector`.
this could be left as-is or changed to `(b - a).cross(c - a)`